### PR TITLE
1308 fe preventing dates longer than 3 months

### PR DESCRIPTION
**Pull request by jekijo**
Fixes #{1308}

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

While still able to click on and select dates outside of the available range when using the disabledDays arg, they are now greyed out and return no data when selected.

### CODE REVIEW

File: server/api/code/lacity_data_api/models/service_request.py [hunk 1]
```diff
@@ -162,6 +162,7 @@ async def get_open_request_counts():
     return result
 
 
+@cached(alias="default")
```
Comment (nichhk) [inline server/api/code/lacity_data_api/models/service_request.py:165]: Please delete this change from the PR

**Review by nichhk — CHANGES_REQUESTED** [review]
Looks great, thanks!

Could you attach a screenshot showing how the calendar looks now?

**Comment by jekijo** [comment]
Screen Shot 2022-09-08 at 7 41 57 PM
Screen Shot 2022-09-08 at 7 42 22 PM
Screen Shot 2022-09-08 at 7 42 45 PM

File: server/api/code/lacity_data_api/models/service_request.py (hunk 1 above)
Comment (nichhk) [reply server/api/code/lacity_data_api/models/service_request.py:165]: Friendly ping

**Review by nichhk — CHANGES_REQUESTED** [review]
Thanks for the screenshots! Looks great. I left another comment on a comment from the first round that might've been missed.

**Review by nichhk — APPROVED** [review]
Looks great, thanks Killen! Feel free to merge when you are ready.

**Comment by melissahermes99** [comment]
It's looking great! Would it be possible to cut the calendar length to align with the line on the dropdown? The calendar color also might blend in too much with the rest of the menu. I also just realized that we no longer have a view charts button on the map menu. I'm not sure if we did that intentionally. Let me run that by Elizabeth to confirm that decision and ask her thoughts on the calendar color.

**Comment by melissahermes99** [comment]
image
Would this be possible to create?

**Pull request merged**